src/vivarium_census_prl_synth_pop/components/population.py

```diff
@@ -331,8 +320,8 @@ def initialize_newborns(self, pop_data: SimulantData) -> None:
         new_births["born_in_us"] = True
 
         # add first and middle names
-        names = self.name_generator.generate_first_and_middle_names(new_births)
-        new_births = pd.concat([new_births, names], axis=1)
+        new_births["first_name_id"] = new_births.index
```

Comment:
Does this value increment from 0 or from the last index value in the pop table? If the latter, this looks fine

Reply:
Last index in the pop table.